core/api/src/services/bria/index.types.d.ts

```diff
@@ -16,6 +16,13 @@ type PayoutMetadata = {
   }
 }
 
+type PayoutSpeedRes = {
```

Comment:
this must be part of domain, if PayoutSpeed cant be used please use PayoutQueue (only at domain level)